config/owasp/suppressions.xml

```diff
@@ -91,4 +91,11 @@
     <packageUrl regex="true">^pkg:maven/org\.postgresql/postgresql@.*$</packageUrl>
     <cve>CVE-2022-21724</cve>
   </suppress>
+  <suppress until="2022-05-01">
+    <notes><![CDATA[
+   file name: spring-beans-5.3.16.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.springframework/spring-beans@.*$</packageUrl>
+    <vulnerabilityName>CVE-2022-22965</vulnerabilityName>
+  </suppress>
```

Comment:
This is a false positive warning because we deploy as jar files and not war files.